app/services/analytics_events.rb

```diff
@@ -921,6 +926,82 @@ def idv_start_over(
     )
   end
 
+  # @param [String] controller
+  # @param [Boolean] user_signed_in
+  # Authenticity token is invalid
```

Comment:
I believe this is the CSRF token, could maybe add that here just for findability
```suggestion
  # Authenticity token (CSRF) is invalid
```